lambdas/answer/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandler.java

```diff
@@ -40,7 +40,8 @@
     private static final String HEADER_SESSION_ID = "session-id";
     private static final String ERROR_KEY = "\"error\"";
     private static final String POST_ANSWER = "post_answer";
-    private final ObjectMapper objectMapper;
```

Comment:
We shouldn't initialise the objectMapper like this as it means we will not be able to make assertions against a mock ObjectMapper instance to verify the code is working as expected & test coverage will be negatively impacted